dao/src/test/java/cz/idomatojde/UserTests.java

```diff
@@ -9,6 +9,7 @@
 import org.springframework.test.context.TestExecutionListeners;
 import org.springframework.test.context.testng.AbstractTestNGSpringContextTests;
 import org.springframework.test.context.transaction.TransactionalTestExecutionListener;
+import org.testng.Assert;
```

Comment:
I would not add another way of asserting things, should be changed to be consistent with the rest of the file.